src/java.compiler/share/classes/javax/lang/model/util/SimpleElementVisitorPreview.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2019, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
general, please check the copyright years

Reply:
Starting with 2019 was intention as a copied the corresponding FooVisitor14 file into FooVisitorPreview and then began editing.

Reply:
ok